yolapy/models/site.py

```diff
@@ -7,11 +7,21 @@ class Site(object):
     """Represents a Site resource on the Yola API."""
 
     def __init__(self, **kwargs):
+        self.fields = kwargs
         for key, val in iteritems(kwargs):
             setattr(self, key, val)
 
+    def __eq__(self, other):
+        return self.fields == other.fields
+
     @classmethod
     def get(cls, site_id):
         """Get a site from the Yola API."""
         site_attributes = Yola().get_site(site_id)
-        return Site(**site_attributes)
+        return cls(**site_attributes)
+
+    @classmethod
+    def list(cls, **filters):
+        """Get a list of sites from the Yola API."""
```

Comment:
Yup, that will work too.  But usually, we just wrap with `PaginatedAPIIterator` existing service methods (which already accept page control params)  and do that on model level.